docs/resources/Guild.md

```diff
@@ -9,49 +9,48 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
+| vanity_url_code                | ?string                                                                                                      | the vanity url code for the guild                                                                                                                                      |
+| description                    | ?string                                                                                                      | the description of a guild                                                                                                                                             |
+| banner                         | ?string                                                                                                      | [banner hash](#DOCS_REFERENCE/image-formatting)                                                                                                                        |
+| premium_tier                   | integer                                                                                                      | [premium tier](#DOCS_RESOURCES_GUILD/guild-object-premium-tier) (Server Boost level)                                                                                   |
+| premium_subscription_count?    | integer                                                                                                      | the number of boosts this guild currently has                                                                                                                          |
+| preferred_locale               | string                                                                                                       | the preferred [locale](#DOCS_REFERENCE/locales) of a Community guild; used in server discovery and notices from Discord, and sent in interactions; defaults to "en-US" |
+| public_updates_channel_id      | ?snowflake                                                                                                   | the id of the channel where admins and moderators of Community guilds receive notices from Discord                                                                     |
+| max_video_channel_users?       | integer                                                                                                      | the maximum amount of users in a video channel                                                                                                                         |
+| max_stage_video_channel_users? | integer                                                                                                      | the maximum amount of users in a stage video channel                                                                                                                   |
```

Comment:
is this (or `max_video_channel_users`) ever not present? I believe they both just have defaults, even if it isn't a community server.

Reply:
https://github.com/discord/discord-api-docs/issues/2034#issuecomment-700531943

Reply:
fair enough....it's been 2 years so im gunna say it's fine to mark them as defined unless there are other objections